app/assets/stylesheets/application/base/fonts.scss

```diff
@@ -10,8 +10,32 @@ http://emojisymbols.com/
   font-style: normal;
 }
 
-
 @font-face {
   font-family: 'NotoSansJP';
   src:font-url('NotoSansJP-Thin.otf') format('opentype');
-}
\ No newline at end of file
+}
+
+@font-face {
+  font-family: 'icomoon';
+  font-style: normal;
+  font-weight: normal;
+  src:font-url('icomoon.woff') format('woff');
+}
+
+[class^="icon-"], [class*=" icon-"] {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings